### PR TITLE
search food & nutrient component

### PR DESCRIPTION
**Pull request by vthtuy**

### CODE REVIEW

File: src/app/searchcomponent/searchcomponent.component.ts [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+ 
+
+@Component({
+  selector: 'app-searchcomponent',
+  templateUrl:'./searchcomponent.component.html',
+  styleUrls: ['./searchcomponent.component.css']
+})
+export class SearchcomponentComponent {
+
+  SUB_SEARCH = environment.mwaNodeServer + '/api/search/food'
```
Comment (vnts7) [inline src/app/searchcomponent/searchcomponent.component.ts]: remove environment.mwaNodeServer

File: src/index.html [hunk 2]
```diff
@@ -7,6 +7,9 @@
 
   <meta name="viewport" content="width=device-width, initial-scale=1">
   <link rel="icon" type="image/x-icon" href="favicon.ico">
+  <link href="https://fonts.googleapis.com/css?family=Roboto:300,400,500" rel="stylesheet">
+  <link href="https://fonts.googleapis.com/icon?family=Material+Icons" rel="stylesheet">
```
Comment (vnts7) [inline src/index.html]: remove

File: src/environments/environment.prod.ts [hunk 3]
```diff
@@ -1,3 +1,4 @@
 export const environment = {
-  production: true
+  production: true,
+  mwaNodeServer: 'http://localhost:1234'
```
Comment (vnts7) [inline src/environments/environment.prod.ts]: dont use static host

File: src/app/app.module.ts [hunk 4]
```diff
@@ -26,15 +34,23 @@ import { LoginComponent } from './login/login.component';
...
-    
+    LoginComponent
   ],
+  
   imports: [
     BrowserModule,
     BrowserAnimationsModule,
     HttpClientModule,
+    BrowserAnimationsModule,
+    MatAutocompleteModule, MatFormFieldModule,
```
Comment (vnts7) [inline src/app/app.module.ts]: duplicated

File: angular.json [hunk 5]
```diff
@@ -84,6 +84,7 @@
               "src/assets"
             ],
             "styles": [
+              "./node_modules/@angular/material/prebuilt-themes/indigo-pink.css",
```
Comment (vnts7) [inline angular.json]: remove

File: src/environments/environment.prod.ts (hunk 3 above)
Comment (vthtuy) [reply src/environments/environment.prod.ts]: this is configuration, not static.
tell me how to get backend server url in angular?

File: src/app/app.module.ts (hunk 4 above)
Comment (vthtuy) [reply src/app/app.module.ts]: will fix

File: src/app/searchcomponent/searchcomponent.component.ts (hunk 1 above)
Comment (vthtuy) [reply src/app/searchcomponent/searchcomponent.component.ts]: how to get backend server host?

File: src/index.html (hunk 2 above)
Comment (vthtuy) [reply src/index.html]: https://github.com/vnts7/mwa-prj/pull/6/commits/3ed18760138de0c7efb0518a9de417177823d10a

File: src/app/app.module.ts (hunk 4 above)
Comment (vthtuy) [reply src/app/app.module.ts]: https://github.com/vnts7/mwa-prj/pull/6/commits/3ed18760138de0c7efb0518a9de417177823d10a

File: angular.json (hunk 5 above)
Comment (vthtuy) [reply angular.json]: https://github.com/vnts7/mwa-prj/pull/6/commits/3ed18760138de0c7efb0518a9de417177823d10a

File: src/environments/environment.prod.ts (hunk 3 above)
Comment (vthtuy) [reply src/environments/environment.prod.ts]: done https://github.com/vnts7/mwa-prj/pull/6/commits/4eb9fdb6141c2aab63f56ec7a7df62b8a66f5ec9

File: src/app/searchcomponent/searchcomponent.component.ts (hunk 1 above)
Comment (vthtuy) [reply src/app/searchcomponent/searchcomponent.component.ts]: done https://github.com/vnts7/mwa-prj/pull/6/commits/4eb9fdb6141c2aab63f56ec7a7df62b8a66f5ec9

**Pull request merged**